### DISCUSSION TITLE
eslint-disable in MSX

### DISCUSSION THREAD

**Discussion by isacvale** [mdx-js/mdx#1771] [Q&A]
Hello! I am working on documenting some react components with Storybook docs, using MDX. `eslint-mdx` allowed me to lint the JSX components, but it doesn't seem to recognise `eslint-disable-line` and similar - it completely ignores those. In this particular project, the warnings eslint raise block the CI, so I cannot integrate linting MDX to the pipeline.

I haven't seen this "eslint-disable" issue addressed in the documentation or in discussions, so I was wondering if there are any extra steps I must take to get it working, or if somebody could point me to the right documentation? Thanks a lot.

[the remark this answers is not in the document]
**Reply by JounQin** [reply]
A reproduction is required.

[the remark this answers is not in the document]
**Reply by jcstein** [reply]
I'm actually experiencing the same thing here. how do i disable linting for `.mdx` files (or portions of them)? https://github.com/celestiaorg/docs/actions/runs/3500889789/jobs/5864000218#step:6:15
